pkg/oc/graph/kubegraph/edges.go

```diff
@@ -25,6 +25,8 @@ const (
 	ExposedThroughServiceEdgeKind = "ExposedThroughService"
 	// ManagedByControllerEdgeKind goes from Pod to controller when the Pod satisfies a controller's label selector
 	ManagedByControllerEdgeKind = "ManagedByController"
+	// OwnedByControllerEdgeKind indicates that an object is owned by a controller (and owner reference has controller=true)
+	OwnedByControllerEdgeKind = "OwnedByController"
```

Comment:
Does this edge end up being added anywhere?

Reply:
no, leftover, good catch!